docs/guide/02_self-hosting/docker.md

```diff
@@ -185,3 +187,59 @@ volumes:
 ```shell
 docker-compose up front
 ```
+
+### Storage
+
+By default, Lago use the internal storage of the container.
+You can customize it by settings different environment variables.
+
+We actually support : 
+  - AWS S3
+  - AWS S3 Compatibles Endpoints
+  - Google Cloud Service Cloud Storage
+
+#### AWS S3
+
+You have to set those variables to use AWS S3.
+
+|Name|Description|
+|--|--|
+|`LAGO_USE_AWS_S3`|Set to `"true"` if you want to use AWS S3|
+|`LAGO_AWS_S3_ACCESS_KEY_ID`|AWS S3 Credentials Access Key Id|
+|`LAGO_AWS_S3_SECRET_ACCESS_KEY`|AWS S3 Credentials Secret Access Key|
+|`LAGO_AWS_S3_REGION`|AWS S3 Region|
+|`LAGO_AWS_S3_BUCKET`|AWS S3 Bucket|
+
+#### AWS S3 Compatible Endpoints
+
+You have to set those variables to use AWS S3 Compatible Endpoints.
+
+|Name|Description|
+|--|--|
+|`LAGO_USE_AWS_S3`|Set to `"true"` if you want to use AWS S3 Compatible Endpoints|
+|`LAGO_AWS_S3_ENDPOINT`|AWS S3 Compatible Endpoint|
+|`LAGO_AWS_S3_ACCESS_KEY_ID`|AWS S3 Credentials Access Key Id|
+|`LAGO_AWS_S3_SECRET_ACCESS_KEY`|AWS S3 Credentials Secret Access Key|
+|`LAGO_AWS_S3_BUCKET`|AWS S3 Bucket|
+
+#### Google Cloud Service Cloud Storage
+
+You have to set those variables to use GCS Cloud Storage.
+
+|Name|Description|
+|--|--|
+|`LAGO_USE_GCS`|Set to `"true"` if you want to use GCS Cloud Storage|
+|`LAGO_GCS_PROJECT`|GCS Project name|
+|`LAGO_GCS_BUCKET`|GCS Bucket name|
+
+Into the `docker-compose.yml` file, you have to uncomment the lines, and pass the correct GCS credentials json file to it.
```

Comment:
In the `docker-compose.yml` file, you must uncomment the lines and pass the correct GCS credentials json file.